ui/Screen/Profile/Tracking.svelte

```diff
@@ -83,7 +106,10 @@
           <div
             class="undiscovered-project"
             out:fade|local={{ duration: FADE_DURATION }}>
-            <Urn urn={project.urn} showCopyOnlyOnHover />
+            <div>
+              <p class="project-name typo-text-bold">{project.metadata.name}</p>
+              <Urn urn={project.urn} showCopyOnlyOnHover />
```

Comment:
The string that should be copied to clipboard should be formatted like this:
`@hyndb5gs95gwtsf37tncz4ag3wqrg4ejw3qqga6x1srw9jp8jw59d6.git#rudolfs`.

https://github.com/radicle-dev/radicle-upstream/issues/840

Reply:
Decided to do this when we tackle #840